src/components/main/EditMemberPreferences.tsx

```diff
@@ -27,12 +27,12 @@ import {
   PREFERENCES_SAVE_BUTTON_ID,
 } from '@/config/selectors';
 
-type EditPreferencesProp = {
-  onCancel: () => void;
+type oncloseProp = {
```

Comment:
This should not have been changed. 
```suggestion
type EditPreferencesProp = {
```